.github/workflows/build.yml

```diff
@@ -252,18 +246,24 @@ jobs:
           </toolchains>
           EOF
       - name: Install Java client
-        run: mvn --also-make --batch-mode --no-transfer-progress --projects clickhouse-http-client -DskipTests install
+        run: mvn --also-make --batch-mode --no-transfer-progress -DskipTests install
+      - name: Generate JWT
+        env:
+          JWT_K_PEM: ${{ secrets.JWT_K_PEM }}
+        run: |
+          npm install jsonwebtoken &&
+          echo "CLIENT_JWT=$(node jwt-gen.js)" >> "$GITHUB_ENV"
       - name: Test http client
         env:
-          CLICKHOUSE_CLOUD_HOST: ${{ secrets.INTEGRATIONS_TEAM_TESTS_CLOUD_HOST_SMT }}
-          CLICKHOUSE_CLOUD_PASSWORD: ${{ secrets.INTEGRATIONS_TEAM_TESTS_CLOUD_PASSWORD_SMT }}
```

Comment:
we need to roll back to INTEGRATIONS_TEAM_TESTS_CLOUD_HOST_SMT. Only this instance has JWT feature enabled

Reply:
done.